packages/@ourworldindata/grapher/src/sourcesTab/SourcesTab.tsx

```diff
@@ -138,10 +142,18 @@ export class SourcesTab extends React.Component<{
                                 <td>{column.unitConversionFactor}</td>
                             </tr>
                         ) : null}
-                        {attribution ? (
+                        {citationProducer.length > 0 ? (
                             <tr>
                                 <td>Data published by</td>
-                                <td>{attribution.join(", ")}</td>
+                                <td>
+                                    <ul>
+                                        {citationProducer.map(
```

Comment:
The bullet point looks a bit weird if there's only one citation (see http://staging-site-fix-repetitions-in-sources-tab/admin/variables/791804). Perhaps it'd be better to only show bullet points with 2+ citations?